datadog/api/monitors.py

```diff
@@ -104,3 +104,21 @@ def unmute_all(cls):
         :returns: Dictionary representing the API's JSON response
         """
         return super(Monitor, cls)._trigger_class_action('POST', 'unmute_all')
+
+    @classmethod
+    def search_monitors(cls, **params):
+        """
+        Search monitors
+
+        :returns: Dictionary representing the API's JSON response
+        """
+        return super(Monitor, cls)._trigger_class_action('GET', 'search', params=params)
+
+    @classmethod
+    def search_monitor_groups(cls, **params):
+        """
+        Search monitor monitor groups
```

Comment:
Nit extra `monitor` in comment

Reply:
Other nits: could you add periods at end of the both method descriptions?